crates/sui-indexer/src/apis/governance_api_v2.rs

```diff
@@ -33,6 +33,22 @@ impl GovernanceReadApiV2 {
         Self { inner }
     }
 
+    pub async fn get_validators_apy(&self) -> Result<ValidatorApys, IndexerError> {
```

Comment:
This was moved from below (`impl GovernanceReadApiServer for GovernanceReadApiV2`) and it is called in that function. The reason for this is that this needs to be public to be reused within the GraphQL.